portal-api/src/v1/users/validation/rules/select-at-least-one-bank.test.js

```diff
@@ -0,0 +1,27 @@
+const selectAtLeastOneBank = require('./select-at-least-one-bank');
+
+describe('selectAtLeastOneBank', () => {
+  const selectAtLeastOneBankError = [
+    {
+      bank: {
+        order: '4',
+        text: 'Bank is required',
+      },
+    },
+  ];
+
+  const testCases = [{ description: 'when no bank is provided', change: { bank: '' } }];
```

Comment:
Same as above, please also test Bank with `*` value.

Reply:
Added test for Bank.